### PR TITLE
[Placeholder] Simplify Command

### PR DESCRIPTION
**Pull request by koinzhang**
## Description

- Simplifying the Random Placeholder Image Command

## Screencast

placeholder-1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/placeholder/package.json [hunk 1]
```diff
@@ -107,41 +107,85 @@
...
-          ],
-          "required": false,
-          "default": "every"
         }
       ]
     },
     {
-      "name": "random-placeholder-image",
-      "title": "Random Placeholder Image",
+      "name": "random-placeholder",
```
Comment (pernielsentikaer) [inline extensions/placeholder/package.json:114]: Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

File: extensions/placeholder/package.json [hunk 2]
```diff
@@ -13,16 +13,16 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "search-placeholder-images",
-      "title": "Search Placeholder Images",
+      "name": "search-placeholders",
```
Comment (pernielsentikaer) [inline extensions/placeholder/package.json:16]: Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by koinzhang** [comment]
Hi @pernielsentikaer Thanks for the tip, I didn't really know what the command name did before, so I changed the command name for more concise use.
I have now reverted back to the original command name, as this name change had no real effect on the command.
However, in other extension updates, because I refactored the actual logic of the command, I had to change the command name to make it more consistent with what the command actually does, e.g. https://github.com/raycast/extensions/pull/12833

**Comment by pernielsentikaer** [comment]
The name is not really that important since the end user never sees it, it's more annoying for them to loose hotkeys and aliases 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**